src/main/kotlin/org/parchmentmc/scribe/util/desc-index-utils.kt

```diff
@@ -30,12 +30,14 @@ import com.intellij.psi.PsiParameter
 val PsiParameter.jvmIndex: Byte
     get() {
         val containingMethod = this.declarationScope as? PsiMethod ?: return -1
-        val thisIndex = containingMethod.parameterList.getParameterIndex(this)
+        val thisIndex = containingMethod.getParameterIndexOffset() + containingMethod.parameterList.getParameterIndex(this)
         return containingMethod.iterateJvmIndices { curIndex, curJvmIndex -> if (curIndex == thisIndex) curJvmIndex else null } ?: -1
     }
 
 fun PsiMethod.getParameterByJvmIndex(jvmIndex: Byte): PsiParameter? {
-    return iterateJvmIndices { curIndex, curJvmIndex -> if (curJvmIndex == jvmIndex) this.parameterList.getParameter(curIndex) else null }
+    return iterateJvmIndices { curIndex, curJvmIndex -> if (curJvmIndex == jvmIndex) {
+        this.parameterList.getParameter(curIndex - this.getParameterIndexOffset())
+    } else null }
```

Comment:
Formatting looks a little weird here. Since you added a new line to the closure, I would recommend inserting a newline after `->` and `else null`.